apis/goharbor.io/v1beta1/harbor_types.go

```diff
@@ -374,6 +374,16 @@ type CoreComponentSpec struct {
 
 	// +kubebuilder:validation:Optional
 	Metrics *harbormetav1.MetricsSpec `json:"metrics,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:default=50
+	MaxIdleConnections *int32 `json:"maxIdleConnections,omitempty"`
```

Comment:
Why is this set here and not in `HarborDatabaseSpec`?

Reply:
This is passed to `Core` and eventually Harbor core deployment to configure the max connections between `core` pods and the db, ultimately goes to https://sourcegraph.com/github.com/goharbor/harbor@cbb211e6702517fddcc8b9a7341e8a3479211ca7/-/blob/src/common/dao/pgsql.go?L99.

This is different from the configuring DB's server-side max connections.

Therefore I think this should be in harbor_types instead of in `HarborDatabaseSpec`. Thoughts?